packages/@o3r/rules-engine/testing/jest.config.it.js

```diff
@@ -1,11 +1,23 @@
 const { dirname } = require('node:path');
-const getJestConfig = require('../../../../jest.config.it').getJestConfig;
+const getJestProjectConfig = require('../../../../jest.config.ut').getJestProjectConfig;
```

Comment:
this doesn't feel right
maybe we can add a comment to explain why this would be needed ?

Reply:
good point. I parametrized the one from the root so I can pass the angularSetup boolean as true.